infrastructure_source_code.md

```diff
@@ -0,0 +1,51 @@
+# DataWorks Open Source Infrastructure Source Code Policy
+
+## Purpose
+
+This document sets out local policy, for the Department for Work and Pensions (DWP) DataWorks team, on what types of information can and cannot be included in infrastructure code that is made publicly available.
+
+It is expected that, at some point, this document will be superceded by a DWP-wide open-source infrastructure code policy which encompasses all DWP public GitHub repositories. Some useful [DWP guidance on creating open source products](https://confluence.service.dwpcloud.uk/pages/viewpage.action?title=Guidance+on+creating+open+source+products&spaceKey=EN) currently exists, but it has not yet been made open-source.
+
+## Scope
+
+This policy covers infrastructure code. Please refer to the [index](README.md#index) for links to other related polices.
+
+## General Approach
+
+Infrastructure code should be made open wherever it is appropriate to do so:
+
+1. Code is more readily accessible to a broad target audience, so that they can be more easily used for their intended purpose;
```

Comment:
"so that they can be more easily used for their intended purpose" might read better as "so that this can be more easily used for its intended purpose"